src/som/primitives/StringPrims.java

```diff
@@ -150,6 +150,7 @@ protected boolean isTaggedWithIgnoringEagerness(final Class<?> tag) {
       }
     }
 
+    @TruffleBoundary
```

Comment:
Hmm, why did you add a boundary here?
The code looks fine. It should compile, no?

Reply:
It's probably a relict of changes that are not in this PR.
If I have an issue with this at some point , I'll solve it differently.

Reply:
I just wrote a microbenchmark to see whether and how it compiles.
And I looked at the IGV graphs. It compiles surprisingly good, despite the three times bounds checking (SOMns, Java string, char array).

So, I would expect it to be safe to remove the boundary here.